modules/ROOT/pages/_partials/push/1_obtaining_firebase_cloud_messaging_credentials.adoc

```diff
@@ -0,0 +1,22 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+[id='obtaining_firebase_cloud_messaging_credentials-{context}']
+= Obtaining Firebase Cloud Messaging Credentials
+
+This procedure describes how to obtain Firebase Cloud Messaging Credentials.
+
+.Prerequisites
+
+* Before the Android application is able to receive the notifications, you must set up access to Firebase Cloud Messaging:
+
+ As with iOS, you are only able to test out your push notifications in a real device.
+ The following credentials are necessary to set up Firebase Cloud Messaging for your app:
+ * the **Server key**
+ * the **Sender ID**
+ * the `google-services.json` file containing the credentials required to connect your app to Firebase and Google services.
+
+.Procedure
+
+. From the _Project Settings_ screen, switch to the _Cloud Messaging_ tab, where you can find the **Server key** and **Sender ID** (known in GCM as **Project Number**). There is also a _Legacy server key_ but it should not be used for new projects.
```

Comment:
we shouldn't be documenting firebase website layout: https://github.com/aerogear/mobile-docs/issues/388